services/head_tracker.go

```diff
@@ -87,6 +86,7 @@ func (ht *HeadTracker) Start() error {
 	go ht.listenForNewHeads()
 	<-ht.subscriptionSucceeded
 
+	ht.fastForwardHeadFromEth()
```

Comment:
My understanding of this change: on startup we subscribe to new headers from the local last known header, and then fast forward to the globally last known header, or at least however up to date the latest ethereum node is(call it the fast-forwarded-to header).

It seems like there is a possibility that we will get two notifications for the fast-forwarded-to header. That is not a problem though, because they will be of the same height, so the second notification is ignored. Is that correct?